game_eggs/minecraft/java/forge/curseforge-generic/egg-curseforge-generic.json

```diff
@@ -29,17 +29,17 @@
             "container": "openjdk:8-jdk-slim",
             "entrypoint": "bash"
         }
     },
     "variables": [
         {
             "name": "Modpack Project ID",
-            "description": "The modpack project ID from the curseforge site on the pack page.\r\n\r\n(Ex. https:\/\/www.curseforge.com\/minecraft\/modpacks\/bofa-mods ID is 375152)",
+            "description": "The modpack project ID from the curseforge site on the pack page.\r\nOr 'zip' if installing from an uploaded server.zip file.\r\n(Ex. https:\/\/www.curseforge.com\/minecraft\/modpacks\/bofa-mods ID is 375152)",
             "env_variable": "MODPACK_ID",
-            "default_value": "",
+            "default_value": "zip",
```

Comment:
I'd prefer not defaulting to "zip" only because people will skip this and not get notified that it's required when they try to create the server. This could cause installs to fail for people that aren't paying attention.

Reply:
Yea that makes sense, I totally agree.
Just pushed new commit reverting it.